getsolus/packages
# ibus-bamboo: add at 0.8.4

**Summary**
- add ibus-bamboo at v0.8.4 (https://github.com/BambooEngine/ibus-bamboo/releases/tag/v0.8.4-RC1)
- known issue with Wayland native (see BambooEngine/ibus-bamboo#44)



**Test Plan**

- Install ibus and run "IBus Preferences"
- Add Vietnamese - Bamboo from Input Method tab
- Add "Keyboard Layouts" plugin/widge to see the input switch icon
- Play around with language switch and many options from IBus Bamboo popup menu
- Focus on text input from browser, terminal or other app
- Type "xin chào các bạn"

**Checklist**
